aiodnsbrute/cli.py

```diff
@@ -317,6 +329,22 @@ def main(**kwargs):
         query=kwargs.get("query"),
     )
 
+    if recursive:
+        # copy it to not modify the collection during iteration
+        results_copy = results
```

Comment:
To achieve the goal in the comment, this would need to be `results_copy = results.copy()`

Reply:
New to process - please forgive me. Do I now reclone my fork, make the mod, and republish a pull request?

Reply:
ya, that sounds like the rough process @IAmStoxe .  Here is the precise steps to follow.
```
git checkout git@github.com:IAmStoxe/aiodnsbrute.git
cd aiodnsbrute
# edit files
git commit --amend -a -m "Added recursion option"
git push --force
```

Reply:
Done :)

Reply:
Nicely done @IAmStoxe.

Since you have been able to do that, another improvement is to undo all of the whitespace changes you have made in this patch *above* your change.

See https://github.com/blark/aiodnsbrute/pull/6.patch

`+@click.option(` should be the first modification in that patch, as that is where your improvement begins.
The preceding changes are undesirable.  Removing them will make it easier for the patch to be reviewed, and less effort for the maintainer to merge the patch.

Reply:
@jayvdb Thank you for that! I am absolutely miserable at diffing apparently, but I think I ended with the right stuff.

Thanks for the tip!

Reply:
Looks quite good now.

Reply:
Thanks again for the help @jayvdb - Very welcoming and understanding to the noob questions.